core/macros/src/main/scala/io/udash/macros/PropertyMacros.scala

```diff
@@ -380,8 +377,13 @@ class PropertyMacros(val ctx: blackbox.Context) extends AbstractMacroCommons(ctx
   def reifyPropertyCreator[A: c.WeakTypeTag]: c.Tree = {
     val tpe = weakTypeOf[A].dealias
 
-    if (!tpe.typeSymbol.isClass) c.abort(c.enclosingPosition, s"Implicit PropertyCreator[$tpe] not found.")
-    else q"new $SinglePropertyCreatorCls[$tpe]"
+    if (!tpe.typeSymbol.isClass) {
+      c.abort(c.enclosingPosition, s"Implicit PropertyCreator[$tpe] not found.")
+    } else if (tpe =:= SeqTpe) {
+      c.abort(c.enclosingPosition,
+        s"Implicit PropertyCreator[Seq[_]] not found. If you use Seq[_] in your model, replace it with Seq[Any]."
```

Comment:
```
scala> typeOf[List[_]] =:= typeOf[List[Any]]
res3: Boolean = false

scala> typeOf[Seq[_]] <:< typeOf[Seq[Any]]
res4: Boolean = true

scala> typeOf[Seq[Any]] <:< typeOf[Seq[_]]
res5: Boolean = true
```

I love Scala

Reply:
The problem here is:
* for `Seq[_]` the compiler does not want to use `PropertyCreatorImplicits.materializeSeq` and takes `PropertyCreatorImplicitsLow.materializeSingle`
* for `Seq[Any]` is uses `PropertyCreatorImplicits.materializeSeq` as expected